src/types/types.ts

```diff
@@ -290,6 +295,7 @@ export interface EdgeCurrencyInfo {
   // Native token information:
   currencyCode: string
   denominations: EdgeDenomination[]
+  tokenMap?: EdgeTokenMap
```

Comment:
Tokens have to move from `EdgeCurrencyInfo` to `EdgeCurrencyConfig`, because that allows custom tokens to get equal treatment. The GUI had a ton of places where people used `currencyInfo.metaTokens` and completely forgot about custom tokens, so we absolutely do *not* want this here. The temptation to write a bug is too strong.

For plugins to list their builtin tokens, the new home is `EdgeCurrencyPlugin.getBuiltinTokens`. This will allow us to achieve another goal, which is to use the info server to update token lists.

Reply:
The issue I ran into that motivated this was `makeOuterPlugin()` in accountbased that would ALWAYS lowercase the contract address when converting to EdgeTokenMap. 

https://github.com/EdgeApp/edge-currency-accountbased/blob/master/src/common/innerPlugin.ts#L78

I needed to prevent that. Since it seemed EdgeTokenMap was the new format, I used that as the trigger to prevent the conversion and leave the EdgeMetaTokens alone. How else would you suggest to prevent `makeOuterPlugin()` from running.

Reply:
Actually, check out this commit in accountbased. I think I figured it out

https://github.com/EdgeApp/edge-currency-accountbased/pull/502/commits/63c8299e7cbc8f740f82479243fb755c6820ccdd

Reply:
The lowercase behavior is intentional and necessary for EVM chains. Rather than do a big refactor, Mathew and I agreed to do a "quick & dirty" upgrade as part of the bundle splitting. Now that build changes are done, I think we need to revisit the hacks with the design Matthew and I really wanted, but didn't have time to do.